int32 order
int32 dataType
uint32 numDimensions
uint32[] dims
uint32[] strides
string name
uint32 offset
